### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by saneshark**

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -10,5 +10,5 @@ require "va/api/common"
 # require "pry"
 # Pry.start
```
Comment (ayaleloehr) [inline bin/console]: Nit. Feels duplicative to have this commented out then the same line below it not commented out.

Comment (saneshark) [inline bin/console]: fixed, keep 'em coming :)

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,2 +1,13 @@
+# By default run SimpleCov, but allow an environment variable to disable.
+unless ENV['NOCOVERAGE']
+  require 'simplecov'
+  SimpleCov.start do
+    track_files 'lib/**/*.rb'
```
Comment (ayaleloehr) [inline spec/spec_helper.rb]: Why is this line necessary here but not in the same file in the `va_rx` repo?

Comment (saneshark) [inline spec/spec_helper.rb]: It's a new trick I learned -- va_rx should probably have it too, it ensures that all files are tracked by simplecov. It addresses that frustrating issue where simplecov only tracks coverage for files that have some minimum amount of invocation.

**Comment by ayaleloehr** [comment]
LGTM with one question for my own curiosity. Feel free to merge @saneshark . At some point (post merge is fine) would love a quick review from @kreek as well to get different eyes on this than were on `va_rx`.

**Comment by saneshark** [comment]
@kreek I'll wait for you to merge since I believe this will need to be upmerged with your PR as well before we can CR that PR.

**Comment by kreek** [comment]
LGTM merging 🚀
